src/modm/driver/color/tcs3414.hpp

```diff
@@ -44,82 +44,82 @@ struct tcs3414
-		PULSES_64	= 0b0110,	///< integrate over 64 pulses of sync pin
-		PULSES_128	= 0b0111,	///< integrate over 128 pulses of sync pin
-		PULSES_256	= 0b1000,	///< integrate over 256 pulses of sync pin
+		PULSES_1	= 0,	// integrate over 1 pulses of sync pin
+		PULSES_2	= 1,	// integrate over 2 pulses of sync pin
+		PULSES_4	= 2,	// integrate over 4 pulses of sync pin
+		PULSES_8	= 3,	// integrate over 8 pulses of sync pin
+		PULSES_16	= 4,	// integrate over 16 pulses of sync pin
+		PULSES_32	= 5,	// integrate over 32 pulses of sync pin
+		PULSES_64	= 6,	// integrate over 64 pulses of sync pin
+		PULSES_128	= 7,	// integrate over 128 pulses of sync pin
+		PULSES_256	= 8,	// integrate over 256 pulses of sync pin
 	};
 
 	/// Register addresses
 	enum class
 	RegisterAddress : uint8_t
 	{
-		CONTROL					= 0x00,	///< Control of basic functions
-		TIMING					= 0x01,	///< Integration time control  @see Tcs3414::setIntegrationTime
-		INTERRUPT				= 0x02,	///< Interrupt settings
-		INT_SOURCE				= 0x03,	///< Interrupt source
-		ID						= 0x04,	///< Part number
-		GAIN					= 0x07,	///< Sensitivity settings @see Tcs3414::setGain
-		LOW_THRESH_LOW_BYTE		= 0x08,	///< Low byte of low interrupt threshold
-		LOW_THRESH_HIGH_BYTE	= 0x09,	///< High byte of low interrupt threshold
-		HIGH_THRESH_LOW_BYTE	= 0x0A,	///< Low byte of high interrupt threshold
-		HIGH_THRESH_HIGH_BYTE	= 0x0B,	///< High byte of high interrupt threshold
+		CONTROL					= 0x00,	// Control of basic functions
+		TIMING					= 0x01,	// Integration time control  @see Tcs3414::setIntegrationTime
+		INTERRUPT				= 0x02,	// Interrupt settings
+		INT_SOURCE				= 0x03,	// Interrupt source
+		ID						= 0x04,	// Part number
+		GAIN					= 0x07,	// Sensitivity settings @see Tcs3414::setGain
+		LOW_THRESH_LOW_BYTE		= 0x08,	// Low byte of low interrupt threshold
+		LOW_THRESH_HIGH_BYTE	= 0x09,	// High byte of low interrupt threshold
+		HIGH_THRESH_LOW_BYTE	= 0x0A,	// Low byte of high interrupt threshold
+		HIGH_THRESH_HIGH_BYTE	= 0x0B,	// High byte of high interrupt threshold
 		// Data registers
-		DATA1LOW				= 0x10,	///< Low byte of ADC green channel
-		DATA1HIGH				= 0x11,	///< High byte of ADC green channel
-		DATA2LOW				= 0x12,	///< Low byte of ADC green channel
-		DATA2HIGH				= 0x13,	///< High byte of ADC green channel
-		DATA3LOW				= 0x14,	///< Low byte of ADC green channel
-		DATA3HIGH				= 0x15,	///< High byte of ADC green channel
-		DATA4LOW				= 0x16,	///< Low byte of ADC green channel
-		DATA5HIGH				= 0x17	///< High byte of ADC green channel
+		DATA1LOW				= 0x10,	// Low byte of ADC green channel
+		DATA1HIGH				= 0x11,	// High byte of ADC green channel
+		DATA2LOW				= 0x12,	// Low byte of ADC green channel
+		DATA2HIGH				= 0x13,	// High byte of ADC green channel
+		DATA3LOW				= 0x14,	// Low byte of ADC green channel
+		DATA3HIGH				= 0x15,	// High byte of ADC green channel
+		DATA4LOW				= 0x16,	// Low byte of ADC green channel
+		DATA5HIGH				= 0x17	// High byte of ADC green channel
```

Comment:
Why are you removing all the doxygen?

Reply:
Found it unsightly :') but is easily reversed.
I'll better read about oxygen